.github/workflows/praisonai-pr-review.yml

```
@@ -16,16 +16,20 @@ jobs:
   review:
     runs-on: ubuntu-latest
     if: >
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)